pkgs/applications/networking/p2p/tixati/default.nix

```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Torrent client";
-    homepage = http://www.tixati.com;
+    homepage = "http://www.tixati.com";
```

Comment:
Also, was it your bot which followed the redirect chain when fixed `meta.homepage` ?
It seems that `meta.homepage` was analyzed by the but, but the 301 redirect (`http://www.tixati.com/` -> https://www.tixati.com/) was not followed.

Reply:
@volth the bot doesn't analyze the URL at all it just quotes it. Later, I would like to have it automatically try to upgrade URLs to https.

Reply:
I have a (false) memory of a bot doing that.
But it is not just `s/http/https/` as in this trivial case.

Reply:
sometimes it is `http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html` -> `https://www.oracle.com/java/technologies/javase-jdk8-downloads.html`

Reply:
It's not a false memory! It used to do that for src URLs but the feature got lost at some point!